Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[package]
+name = "mainframer"
+version = "2.1.0"
```

Comment:
In future I'd like to detach version from source code and get it from current git tag, but we didn't do it with Bash so for simplicity I didn't do it in this PR

Reply:
Do you want to open an issue about it?

Reply:
Yeeep, here you go #192